share/pot/term.sh

```diff
@@ -26,28 +26,22 @@ pot-term()
 	local _pname _force
 	_pname=
 	_force=
-	args=$(getopt hvf $*)
-	if [ $? -ne 0 ]; then
-		term-help
-		exit 1
-	fi
 
-	set -- $args
-	while true; do
-		case "$1" in
-		-h)
+	while getopts "hvf" _o; do
```

Comment:
`OPTIND=1` is missing. It's a best practice to always initialize it before call `getopts`

Reply:
Solved.